crates/bitwarden-vault/src/cipher/attachment.rs

```diff
@@ -151,6 +167,18 @@ mod tests {
         Attachment, AttachmentFile, Cipher,
     };
 
+    #[test]
+    fn test_size_name_conversions() {
```

Comment:
Should we also test the `encrypt_with_key`values?

Reply:
Good point, added a test to see that the values get updated